src/modules/outliers/modZScore.js

```diff
@@ -0,0 +1,176 @@
+import i18n from '@dhis2/d2-i18n'
+
+import { PROP_NORMALIZATION_METHOD, PROP_THRESHOLD_FACTOR } from './index'
+import { deNormalizerMap } from './normalization'
+
+export const MODIFIED_Z_SCORE = 'MODIFIED_Z_SCORE'
+
+const MEDIAN_AD_CORRECTION = 0.6745
```

Comment:
Is it worth adding a comment about where these magical numbers come from or is this something straightforward if you're an outlier jedi master?